### ISSUE TITLE
BTPayPalAccountNonce has missing account fields

### ISSUE THREAD

**Issue by derekyu01** [braintree/braintree_ios#1063]
### Braintree SDK Version

6.0.0

### Environment

Production

### Xcode Version

14.3.0

### OS Version & Device

iPhone 14 Pro simulator and Device

### Integration type

Swfit Package Manager

### Development Processor

Apple Silicon (M-series chips)

### Describe the bug

When retrieving BTPayPalAccountNonce from 

```
BTPayPalClient.tokenize(
        _ request: BTPayPalVaultRequest,
        completion: @escaping (BTPayPalAccountNonce?, Error?)
```
BTPayPalAccountNonce is missing account information for all fields. The fields are populated in version 5, but fields are missing when upgrade to version 6.

### To reproduce

1. Create BTPayPalClient and call  
```
public func tokenize(
        _ request: BTPayPalVaultRequest,
        completion: @escaping (BTPayPalAccountNonce?, Error?) -> Void
    )
```
BTPayPalAccountNonce is missing email and account related fields.

### Expected behavior

Account related fields are supposed to be returned and populated in the BTPayPalAccountNonce

**Comment by derekyu01** [comment]
<img width="624" alt="image" src="https://github.com/braintree/braintree_ios/assets/111313856/15b82282-9fe4-4e2b-897b-14ee618f06f3">

**Comment by jaxdesmarais** [comment]
This was released in version [6.2.0](https://github.com/braintree/braintree_ios/releases/tag/6.2.0) of the SDK. Thanks again for reporting this issue! 🚀
